reana_server/config.py

```diff
@@ -30,6 +30,12 @@
 
 SHARED_VOLUME_PATH = os.getenv('SHARED_VOLUME_PATH', '/var/reana')
 
+REANA_URL = os.getenv('REANA_URL', 'reana.io')
+
+CERN_CONSUMER_KEY = os.getenv('CERN_CONSUMER_KEY', 'CHANGE_ME')
+
+CERN_CONSUMER_SECRET = os.getenv('CERN_CONSUMER_SECRET', 'CHANGE_ME')
```

Comment:
What do you think of prefixing this variable and the previous with `REANA_SSO`? For example, `REANA_SSO_CERN_CONSUMER_KEY`.